gems/pending/spec/util/extensions/miq-module_spec.rb

```diff
@@ -18,7 +18,7 @@ module ::TestModule; end
     end
 
     after(:each) do
-      $miq_cache_with_timeout.clear
+      Module.clear_all_cache_with_timeout
```

Comment:
@Fryguy hrm, maybe this `after(:each)` can just be removed since adding the `clear_all_cache_with_timeout` to spec_helper... thoughts?